internal/command/action_test.go

```diff
@@ -240,3 +240,55 @@ testprovider-trailingnewline version 3.2.1
 		So(output, ShouldEqual, expected)
 	})
 }
+
+func TestLocateFileRecurseUp(t *testing.T) {
+	filename := "test.txt"
+	currentDir, _ := os.Getwd()
+	var err error
+
+	Convey("Finds file in current working directory", t, func() {
+		localFilePath := filepath.Join(currentDir, filename)
```

Comment:
This should be a file in a tempdir (always consider the repo dir as pristine)

Reply:
This code was already in the branch from where I started from :-)
Done as requested.